docs/Course Wiki/ECE Course Offerings/ECE340.md

```diff
@@ -0,0 +1,61 @@
+
+## Course Structure
+
+The course is fairly standard with a combination of 11 homeworks, 2 midterms, and a final comprising to total grade. 
+
+Homework assignments are assigned each week and usually consist of three or four problems that may involve drawing band diagrams, derivations, calculations, conceptual explanations, or using an online semiconductor simulation applet. There will be multiple times where certain assumption must be made (temperature independence, neglecting a small concentration, etc) to make ceratin problems solable in a reasonable amount of time. This can be frustrating for students used to solving problems with the exact parameters given to them, but making and justifying these assumptions are part of the course.
+
+Midterm exams for ECE340 are quite imfamous among the 300 level ECE courses. While slight logistical things can vary between semesters (like an equation sheet), in general the tests are 3 questions with several subpart to be solved in 1 hour without a calculator. They are usually even more of a test of time management than ECE210 exams. The strangest quirk about this course is the lack of access to past exams. The testing format also means that there is little similarity to homework problems. This structure makes understanding the concepts taught in ECE340 paramount to success.
+
+## Instructors
+
+Prof. Leburton is the course director. Professors that have taught ECE 340 in the past include Prof. Rakheja, Dragic, He, Zhu, Dallesasse, Bogdanov to name a few. Most professors who teach this course specialize in device physics or nanoelectronics.
+
+## Course Tips
+
+Try not to get discouraged when recieving grades back, this is set up to be a tough course and your overall grade does not fully reflect your ability to work with semiconductors. 
+
+Partial credit on exams is your best friend. If a problem is too challenging, write down as many equations or notes that seem adjacent enough to the problem and the graders may reward it. 
+
+ALWAYS justify an assumption. It is required to complete a problem, but also a good check to make sure the assumption is valid.
+
+## Life After
+
+Not surprisingly, ECE340 is the prerequisite to many courses in the area of solid-state electronics, including ECE441 (Physics and Modeling of Semiconductor Devices), ECE444 (Theory and Fabrication of Integrated Circuits), and ECE482 (Digital IC Design). Taking [ECE342](ECE342.md) (Electronic Circuits) either before, concurrently, or after ECE340 provides a good overview of electronic circuits at both the device and circuit levels.
+
+## Infamous Topics
```

Comment:
For each one, also add how to combat it or something. Ex:
The whole class: Not to worry, the curve will save you.
PN Junctions: Keep track of xyz, do yxe, etc.